### DISCUSSION TITLE
FamilySearch Places: exid-types.json; uri in EXID.TYPE

### DISCUSSION THREAD

**Discussion by albertemmerich** [FamilySearch/GEDCOM#567] [Ideas]
Standard recommends to have a EXID.TYPE paylod which by appending the EXID payload the full url to find the webpage in the refrenced extrenal system. I wonder why this is not done in the examples for FAmily Search systems, like the place system. In the defining exid-types.json we find:
https://gedcom.io/exid-type/FamilySearch-PlaceId

This cannot be expanded to an valid url. It should be:
https://www.familysearch.org/research/places/?focusedId=

??!!

**Comment by tychonievich** [comment]
I've linked this to https://github.com/FamilySearch/GEDCOM-registries/issues/106 as this type is stored there since #576

**Comment by dthaler** [comment]
Should we close this issue and just use https://github.com/FamilySearch/GEDCOM-registries/issues/106 to track?

**Reply by dthaler** [reply]
Agreed in GEDCOM Steering Committee meeting, please use the above issue to discuss
